packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -476,17 +489,17 @@ class OnsetPicker extends React.Component {
           </SelectField>
 
           {granularitySelect}
-        </FieldSet>
 
-        {(this.state.precision !== OnsetUtils.PrecisionOptions.UNKNOWN) && (
-          <FieldSet className={cx('fieldset')}>
-            {ageInput}
-            {ageUnitSelect}
-            {monthSelect}
-            {yearSelect}
-            {dateSelect}
-          </FieldSet>
-        )}
+          {(this.state.precision !== OnsetUtils.PrecisionOptions.UNKNOWN) && (
```

Comment:
Didn't this change need screenshots to be updated?

Reply:
no this has no impact on existing UI. It remains same as before.